src/main/java/org/spongepowered/api/util/SpongeApiTranslationHelper.java

```diff
@@ -62,7 +61,7 @@ private SpongeApiTranslationHelper() {
      * @return The translatable text
      */
     public static Text t(String key, Object... args) {
-        return Texts.of(new ResourceBundleTranslation(key, LOOKUP_FUNC), args);
+        return Text.of(new ResourceBundleTranslation(key, LOOKUP_FUNC), args);
```

Comment:
I know this isn't the right spot, but I cannot comment the declaration above.
`LOOKUP_FUNC` could be replaced with a functional expression.
````java
private static final Function<Locale, ResourceBundle> LOOKUP_FUNC = (locale) ->
        ResourceBundle.getBundle("org.spongepowered.api.Translations", locale);
````